native/NativeSampleApps/test/RestExplorerTest/src/com/salesforce/samples/restexplorer/ServerPickerActivityTest.java

```diff
@@ -285,8 +289,13 @@ public boolean matchesSafely(View view) {
     private static void tapResetButton() {
         openActionBarOverflowOrOptionsMenu(getInstrumentation().getTargetContext());
         try {
-            onView(allOf(withId(android.R.id.title), withText("Reset"), findUiElement()))
-                    .perform(click());
+            onData(CoreMatchers.anything())
```

Comment:
This was little tricky to find, but some new Espresso magic finds the "Reset" options menu item in the new view hierarchy provided by AppCompat. I would like to change `atPosition` to something that scales better if menu items come and go or get renamed.  Does anyone know how to do that in `onData`?